docs/content/tutorial/index.ngdoc

```diff
@@ -129,7 +129,7 @@ Once you have Node.js installed on your machine you can download the tool depend
 npm install
 ```
 
-This command will download the following tools, into the `node_modules` directory:
+This command reads angular-phonecat's project.json file and will download the following tools, into the `node_modules` directory:
```

Comment:
You're right. sorry for the typo.  I fixed it in the branch.

On Mon, Jun 8, 2015 at 2:07 PM, Martin Staffa <notifications@github.com>
wrote:

> In docs/content/tutorial/index.ngdoc
> <https://github.com/angular/angular.js/pull/12040#discussion_r31942048>:
>
> > @@ -129,7 +129,7 @@ Once you have Node.js installed on your machine you can download the tool depend
> >  npm install
> >  ```
> >
> > -This command will download the following tools, into the `node_modules` directory:
> > +This command reads angular-phonecat's project.json file and will download the following tools, into the `node_modules` directory:
>
> I assume you mean package.json?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular.js/pull/12040/files#r31942048>.
>